README.md

```diff
@@ -1,31 +1,37 @@
 # Introduction
 
-snowflake-vcrpy is designed to enhance local development experiences for applications
-built on top of snowflake-connector-python. Libraries like snowpark-python or snowflake-sqlalchemy which depends
-on the connector for http communication can also take advantage of the tool.
+snowflake-vcrpy is a library designed to allow you to record and replay HTTP interactions initiated by snowflake-connector-python for testing purposes. It is a pytest plugin built upon [VCR.py][vcrpy].
+ Libraries like snowpark-python or snowflake-sqlalchemy depending
+on the connector for HTTP interactions can also take advantage of the library.
```

Comment:
Feels like line 3 is much longer than line 4-5 combined

Reply:
they will be rendered the same, but yeah it's good to split line3 for better maintenance.